README.md

```diff
@@ -31,15 +34,14 @@
 - [📝 License](#license)
 
 # 📖 [Hello Microverse!] <a name="about-project"></a>
```

Comment:
[Optional] Please consider removing the brackets [] around your title and in your key features and future features sections🙏. Without these brackets, your readme looks a lot cleaner and more readable. Try it out and see what I mean🫡

Reply:
Thank you, done accordingly.